app/celery/scheduled_tasks.py

```diff
@@ -302,3 +303,15 @@ def populate_monthly_billing():
     start_date, end_date = get_month_start_and_end_date_in_utc(yesterday_in_bst)
     services = get_service_ids_that_need_billing_populated(start_date=start_date, end_date=end_date)
     [create_or_update_monthly_billing(service_id=s.service_id, billing_month=end_date) for s in services]
+
+
+@notify_celery.task(name="run-letter-jobs")
+@statsd(namespace="tasks")
+def run_letter_jobs():
+    job_ids = [job.id for job in dao_get_letter_jobs_by_status(JOB_STATUS_READY_TO_SEND)]
+    notify_celery.send_task(
+        name=QueueNames.DVLA_FILES,
+        args=(job_ids),
```

Comment:
I think we should differentiate between queue names and tasks names. As it's slightly awkward to call `send_task` with the task name being set as the queue name. 

For now we could just put the task name like we do elsewhere and then look at maybe putting this into a `TaskNames` class?

Also, think we should put this task on the `QueueNames.NOTIFY` as we need to make sure the API doesn't read from the `PROCESS_FTP` queue.